### ISSUE TITLE
Share pins on other social networks

### ISSUE THREAD

**Issue by huuvu1109** [seregazhuk/php-pinterest-bot#304]
Hi @seregazhuk ,

Pinterest has one feature which allow one pinner to share a pin on Facebook, Twitter, ... like the first figure . The shared content will contains a link to the shared Pin and allow other Pinners can vote the shared Pin (the second figure). Can the bot do this automatically? I mean to share a pin, get the shared link (created by Pinterest), and another account to use that link to vote. Thanks!

![image](https://user-images.githubusercontent.com/9253443/28531985-b1c8d0d6-7098-11e7-853c-f0b3305f7f85.png)

https://user-images.githubusercontent.com/9253443/28494861-736f0de0-6f3b-11e7-9bb2-a5b5430f3b23.png

**Comment by huuvu1109** [comment]
![image](https://user-images.githubusercontent.com/9253443/28532133-3bb396a0-7099-11e7-855a-0b6483007294.png)

**Comment by seregazhuk** [comment]
Hi, @huuvu1109 It's implemented in v5.5.3

Here is the documentation:

Share a link with a pin where a user can leave his or her reaction on this pin (like or dislike):
```php
$link = $bot->pins->share($pinId);
```

Leave a reaction when you were given a sharing link with a pin. 
The link looks like this: `http://pin.it/cTwZfG_`. When you open it in your browser Pinterest redirects you to 
the following url `https://www.pinterest.de/pin/332703491213209642/sent/?sfo=1&sender=731835145606177283&invite_code=6cb1d66946464b3f9d0084f623c7822b`.
To *react* on this link you need to know a pinId (number after `pin`) and a userId (number after `sender`), who sent you a link:
 
```php
// like
$bot->pins->leaveGoodReaction($pinId, $senderId);
// don't like
$bot->pins->leaveBadReaction($pinId, $senderId);
```
